src/components/CurrentUser/CurrentUser.tsx

```diff
@@ -1,12 +1,74 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { getUserById } from '../../api';
 import './CurrentUser.scss';
 
-export const CurrentUser: React.FC = () => (
-  <div className="CurrentUser">
-    <h2 className="CurrentUser__title"><span>Selected user: 2</span></h2>
+type UserProps = {
+  userId: number,
+  clear: () => void,
+};
 
-    <h3 className="CurrentUser__name">Ervin Howell</h3>
-    <p className="CurrentUser__email">Shanna@melissa.tv</p>
-    <p className="CurrentUser__phone">010-692-6593 x09125</p>
-  </div>
-);
+export const CurrentUser: React.FC<UserProps> = ({
+  userId, clear,
+}) => {
+  const [userName, setUserName] = useState('');
+  const [userEmail, setUserEmail] = useState('');
+  const [userPhone, setUserPhone] = useState('');
```

Comment:
let's store the whole user object inside selectedUser state variable and then render what you need from this object